tests/API/Helpers/State/StateHandlerTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Auth0\Tests\Api\Helpers\State;
+
+use Auth0\SDK\API\Helpers\State\SessionStateHandler;
+use Auth0\SDK\Store\SessionStore;
+
+class StateHandlerTest extends \PHPUnit_Framework_TestCase {
```

Comment:
is the `\` intended?

Reply:
PHP namespace thing, means 'use the global namespace'